osi_common.proto

```diff
@@ -154,6 +154,9 @@ message BaseStationary
 
     /// The relative orientation of the landmark w.r.t its parent frame.
     optional Orientation3d orientation = 3;
+    
+    /// The two dimensional (flat) contour of the object. 
```

Comment:
@AndreasVIRES End-Points of Positions always refer to the middle of the bounding box. Finding out the position is solely possible if relative_ego_origin is set to the rear axis. However relative_ego_origin is solely required for transforming from world to sensor-coordinates or for transforming from sensor-coordinates to the environment model. It does not play any role when staying in the same coordinate system (world or sensor).

We explicitly did not choose the rear axis as reference point, as it plays no role for sensors and just increases the complexity of finding out other points on the vehicle.

I get your point, we should end the discussion here. Further commitment would be helpful.
I am also not willing to explain every reasoning multiple times just due to the lack of participation of others.